pkg/reconciler/source/duck/duck.go

```diff
@@ -211,6 +215,9 @@ func (r *Reconciler) makeEventTypes(ctx context.Context, src *duckv1.Source) []v
 			CeSchema:    schemaURL,
 			Description: description,
 		})
+		if eventType.Spec.Reference.Namespace == "" {
+			eventType.Spec.Reference.Namespace = defaultNamespace
+		}
```

Comment:
If we want to set a namespace, the correct one is `src.Namespace`, assuming default is wrong